python/interpret_community/widget/explanation_dashboard_input.py

```diff
@@ -212,7 +215,9 @@ def on_predict(self, data):
     def _convert_to_list(self, array):
         if issparse(array):
             if array.shape[1] > 1000:
-                raise ValueError("Exceeds maximum number of features for visualization (1000)")
+                raise ValueError("Exceeds maximum number of features for visualization (1000). Please regenerate the"
```

Comment:
@rihorn2 users could run into this even if they don't pass a dataset if their feature array (or even class array?) is too long, right?